catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -182,26 +196,36 @@ describe('determineNextAvailableDate', () => {
     // at 09:30 in London -- it can be fulfilled the next day.
     const date1 = new Date('2021-12-09T10:30:00+0100');
 
-    const result1 = determineNextAvailableDate(date1, [0]);
+    const result1 = determineNextAvailableDate(date1, [0], []);
     expect(result1).toEqual(new Date('2021-12-10T09:30:00Z'));
 
     // Paris is an hour ahead of London, so a request made at 11:30 in Paris is
     // at 19:30 in London -- it can’t be fulfilled the next day.
     const date2 = new Date('2021-12-09T11:30:00+0100');
 
-    const result2 = determineNextAvailableDate(date2, [0]);
+    const result2 = determineNextAvailableDate(date2, [0], []);
     expect(result2).toEqual(new Date('2021-12-11T10:30:00Z'));
 
     // Now run the same tests, but now during British Summer Time when London
     // and UTC are different.
     const date3 = new Date('2022-09-06T10:30:00+0200');
-    const result3 = determineNextAvailableDate(date3, [0]);
+    const result3 = determineNextAvailableDate(date3, [0], []);
     expect(result3).toEqual(new Date('2022-09-07T09:30:00+0100'));
 
     const date4 = new Date('2022-09-06T11:30:00+0200');
-    const result4 = determineNextAvailableDate(date4, [0]);
+    const result4 = determineNextAvailableDate(date4, [0], []);
     expect(result4).toEqual(new Date('2022-09-08T10:30:00+0100'));
   });
+
+  it('accounts for exceptional closure dates', () => {
+    const exceptionalClosure = new Date('2021-12-13'); // Monday
+    const result = determineNextAvailableDate(
+      new Date('2021-12-10 10:30'), // Friday
+      [0],
+      [exceptionalClosure]
+    ); // Sunday
```

Comment:
Why Sunday here? Everything looks logical, just wanting to understand the usual logic for these!

Reply:
You're right, that `// Sunday` shouldn't be there. The result of a copy-paste. I'll get rid of it, thanks.